### PR TITLE
Use gt in mod_ParticipantDomain

### LINKED ISSUE #289: Feature: gt for Participant Domain tables
## Feature Details
Standardize on gt for all tables (specifically the Participant Metric Domain tables here).

### PR DESCRIPTION
**Pull request by jonthegeek**
Eliminates the DT dependency!

## Connected Issues

- Closes #289

### CODE REVIEW

File: R/mod_ParticipantDomain_Server.R [hunk 1]
```diff
@@ -3,50 +3,63 @@
...
     })
-    output$title <- renderText({
-      if (!valid_selection()) {
-        return("Participant Domain")
+
+    rctv_gtObject <- reactive({
+      if (rctv_selectionIsValid()) {
+        df <- rctv_tblData()
+        gtObj <- gt::gt(df) %>%
+          out_gtInteractive(selection_mode = "multiple") %>%
```
Comment (lauramaxwell) [inline R/mod_ParticipantDomain_Server.R:32]: when i select multiple, the app crashes, like below- 
image

**Review by lauramaxwell — CHANGES_REQUESTED** [review]
see comment below re: `selection_mode = "multiple"`

File: R/mod_ParticipantDomain_Server.R (hunk 1 above)
Comment (jonthegeek) [reply R/mod_ParticipantDomain_Server.R:32]: Fixed in latest push.

**Review by lauramaxwell — APPROVED** [review]
nice- looks good to me! no more `DT` 🎉

**Pull request merged**